examples/04_pong/main.rs

```diff
@@ -335,6 +338,20 @@ impl State for Pong {
                 bounce_sfx,
                 score_sfx,
             });
+            // Sets the cursor hidden when the mouse is hovering the pong window.
+            if let Some(mut window_events) = engine
+                .world
+                .res
+                .try_fetch_mut::<EventHandler<WindowModifierEvent>>(0)
+            {
+                window_events.write_single(WindowModifierEvent {
+                    modify: |win| {
+                        win.set_cursor_state(winit::CursorState::Hide)
```

Comment:
How does the cursor become visible again?

Reply:
You need to send a new event to the window. The code is the same, you just replace winit::CursorState::Hide by winit::CursorState::Normal

https://docs.rs/winit/0.8.2/winit/enum.CursorState.html